Updated CMake

ksmith0 opened the pull request:
Changed to ../install from build and amended FindPXI.cmake messages.

Commit:
Changed default install directory.

Changed to ../install from build.

spaulaus reviewed (commented):
This did what it said it did. 

Tested on 
Linux Flavor Arch Linux. 
Kernel : 4.9.6-1-ARCH
gcc version:  6.3.1 20170109 (GCC)
cmake version 3.7.2

Commit:
Corrected messages in FindPXI.cmake

Some messages were not set to STATUS causing them to appear on the
screen in ccmake.

spaulaus reviewed (approved):
Still does what it says. More importantly, it removes that annoying message.
